src/app/pages/common/ix-forms/components/ix-combobox2/ix-combobox2-provider.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { Subject } from 'rxjs';
+import { Option } from '../../../../../interfaces/option.interface';
+
+export interface IxCombobox2Provider {
+  providerUpdater$: Subject<void>;
+  options: Option[];
```

Comment:
How about not having `providerUpdater$` and instead just have `options$` be a stream?